Formula/glances.rb

```diff
@@ -0,0 +1,41 @@
+class Glances < Formula
+  desc "Alternative to top/htop"
+  homepage "https://nicolargo.github.io/glances/"
+  url "https://github.com/nicolargo/glances/archive/v2.11.1.tar.gz"
+  sha256 "446a0ee6e13c0c7ceb4bc0d4868add8a02d5e3ff866de8e880bdb33dce6ab3fc"
+
+  resource "psutil" do
+    url "https://files.pythonhosted.org/packages/e2/e1/600326635f97fee89bf8426fef14c5c29f4849c79f68fd79f433d8c1bd96/psutil-5.4.3.tar.gz"
+    sha256 "e2467e9312c2fa191687b89ff4bc2ad8843be4af6fb4dc95a7cc5f7d7a327b18"
+  end
+
+  depends_on :python
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    resources.each do |r|
+      r.stage do
+        system "python2.7", *Language::Python.setup_install_args(libexec/"vendor")
+      end
+    end
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python2.7", *Language::Python.setup_install_args(libexec)
+
+    bin.install Dir[libexec/"bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+  test do
+    begin
+      read, write = IO.pipe
+      pid = fork do
+        exec bin/"glances", "-q", "--export-csv", "/dev/stdout", :out => write
```

Comment:
Note that this test currently hits the network to determine public IP. There's a `--disable-ip` flag but it doesn't actually prevent the network request. I've opened a [pull request](https://github.com/nicolargo/glances/pull/1216) to fix this.

Reply:
hitting the network is fine